docs/spec.md

```diff
@@ -1384,6 +1384,12 @@ in the `operand` tensor and produces a `result` tensor. More formally,
 * (C5) For all `d` in `axes(operand)`:
   * `dim(operand, d) = 1` or
   * `dim(operand, d) = dim(result, broadcast_dimensions[d])`.
+* (C6) `is_per_tensor_quantized(operand) = is_per_tensor_quantized(result)`.
+* (C7) If `is_per_axis_quantized(result)`:
+  * `quantized_dimension(result) = broadcast_dimensions[quantized_dimension(operand)]`.
+  * `If dim(operand, quantized_dimension(operand)) = 1, then
+    scales(result)[i] = scales(operand)[0] for i in
```

Comment:
`and zero_points(result)[i]] = zero_points(operand)[i]`

Reply:
done